presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/DatabaseShardManager.java

```diff
@@ -365,6 +366,10 @@ private void runCommit(long transactionId, HandleConsumer callback)
                 return;
             }
             catch (DBIException e) {
+                if (isTransactionCacheFullError(e)) {
+                    throw metadataError(e, "The transaction is too large.");
```

Comment:
Remove the period at the end. Error messages aren't full sentences. Also, we can simply the error message to be *"Transaction too large"*